changes/3880.security

```diff
@@ -0,0 +1 @@
+Updated `GitPython` to `3.1.36` to address `CVE-2023-41040`.
```

Comment:
Guessing we may want to backport this to LTM?

Reply:
Yup! Probably won't cherry-pick since it's fairly straight forward to implement it directly.